docs/source/features/file-uploads.md

```diff
@@ -0,0 +1,139 @@
+---
+title: File Uploads
+description: Implementing File Uploads on Apollo Server
+---
+
+File uploads are a requirement for many applications. Apollo Server supports the [GraphQL multipart request spec](https://github.com/jaydenseric/graphql-multipart-request-spec) for uploading files as mutation arguments.
```

Comment:
lets add a pointer to apollo-upload-server here. Something like `mutation arguments, using [apollo-upload-server](https://github.com/jaydenseric/apollo-upload-server)